webroot/css/cake.css

```diff
@@ -382,3 +382,176 @@ pre {
   color: red;
   margin-right: 9px;
 }
+
+
+/* Midori Universal */
```

Comment:
This comment will confuse other users too, note that no other files contain "personal" comments like this.